File: docs/APIs-and-SDKs/SDK-Documentation/getting-started/create-new-context-request/scala/sync.scala
import scala.concurrent.Await
import scala.concurrent.duration._

val units = Map(
  "session_id" -> "5ebf06d8cb5d8137290c4abb64155584fbdb64d8",
  "user_id" -> "123456"
)

val contextDataFuture = sdk.fetchContextDataAsync()

val contextData = Await.result(contextDataFuture, 5.seconds)

val context = sdk.createContextWith(units, contextData)

File: docs/APIs-and-SDKs/SDK-Documentation/getting-started/create-new-context-request/rust/sync.rs
use absmartly_sdk::ContextOptions;

let units = [("session_id", "5ebf06d8cb5d8137290c4abb64155584fbdb64d8")];

let options = ContextOptions {
    publish_delay: 100,
    refresh_period: 0,
    event_logger: None,
};

let mut context = sdk.create_context(units, Some(options)).await?;
